gooddata-sdk/gooddata_sdk/catalog/organization/entity_model/jwk.py

```diff
@@ -0,0 +1,89 @@
+# (C) 2022 GoodData Corporation
+from __future__ import annotations
+
+from typing import List, Optional, Type
+
+import attr
+
+from gooddata_api_client.model.json_api_jwk_in import JsonApiJwkIn
+from gooddata_api_client.model.json_api_jwk_in_attributes import JsonApiJwkInAttributes
+from gooddata_api_client.model.json_api_jwk_in_attributes_content import JsonApiJwkInAttributesContent
+from gooddata_api_client.model.json_api_jwk_in_document import JsonApiJwkInDocument
+from gooddata_sdk.catalog.base import Base
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class CatalogJwkDocument(Base):
+    data: CatalogJwk
+
+    @staticmethod
+    def client_class() -> Type[JsonApiJwkInDocument]:
+        return JsonApiJwkInDocument
+
+    def to_api(self) -> JsonApiJwkInDocument:
+        dictionary = self._get_snake_dict()
+        return self.client_class().from_dict(dictionary, camel_case=False)
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class CatalogJwk(Base):
+    id: str
+    attributes: CatalogJwkAttributes
+
+    @staticmethod
+    def client_class() -> Type[JsonApiJwkIn]:
+        return JsonApiJwkIn
+
+    @classmethod
+    def init(
+        cls,
+        jwk_id: str,
+        attributes: Optional[CatalogJwkAttributes] = None,
+    ) -> CatalogJwk:
+        return cls(id=jwk_id, attributes=attributes)
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class CatalogJwkAttributes(Base):
+    content: Optional[CatalogRsaSpecification] = None
+
+    @staticmethod
+    def client_class() -> Type[JsonApiJwkInAttributes]:
+        return JsonApiJwkInAttributes
+
+    @classmethod
+    def init(
+        cls,
+        content: Optional[CatalogRsaSpecification],
+    ) -> CatalogJwkAttributes:
+        return cls(content=content)
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class CatalogRsaSpecification(Base):
+    alg: Optional[str] = None
+    e: Optional[str] = None
+    kid: Optional[str] = None
+    kty: Optional[str] = None
+    n: Optional[str] = None
+    use: Optional[str] = None
+    x5c: Optional[List[str]] = None
+    x5t: Optional[str] = None
```

Comment:
According to oAPI specification, all of these attributes are mandatory. Please, remove Optional and None as default.